model_compression_toolkit/core/common/quantization/filter_nodes_candidates.py

```diff
@@ -81,5 +114,8 @@ def filter_node_candidates(node: BaseNode) -> List[CandidateNodeQuantizationConf
 
         for c in filtered_candidates:
             c.weights_quantization_cfg.weights_n_bits = FLOAT_BITWIDTH
+            c.weights_quantization_cfg.weights_quantization_method = QuantizationMethod.POWER_OF_TWO
+
+        final_candidates = _filter_bit_method_dups(filtered_candidates)
```

Comment:
why you are using **_filter_bit_method_dups** in 2 places instead one instance before the final return? I guess it's not suppose to effect the single_dummy_candidate.

Reply:
there is another case that neither of the "if" statements is executed, then an outer call to _filter_bit_method_dups would run on the entire candidates list.
It's not supposed to affect the outcome but I wouldn't want to run something that is not necessary